js/content/community.js

```diff
@@ -571,17 +547,26 @@ let ProfileHomePageClass = (function(){
             htmlstr += CommunityCommon.makeProfileLink("custom", link, name, iconType, icon);
         }
 
-        // profile permalink
-        if (SyncedStorage.get("profile_permalink")) {
-            let imgUrl = ExtensionResources.getURL("img/clippy.svg");
-            htmlstr +=
-                `<div id="es_permalink_div" class="profile_count_link">
-					<span class="count_link_label">${Localization.str.permalink}</span>
-					<div class="es_copy_wrap">
-						<input id="es_permalink" type="text" value="https://steamcommunity.com/profiles/${steamId}" readonly />
-						<button id="es_permalink_copy"><img src="${imgUrl}" /></button>
-					</div>
-				</div>`;
+        // profile steamid
+        if (SyncedStorage.get("profile_steamid")) {
+            let dropdown = document.querySelector("#profile_action_dropdown .popup_body.popup_menu");
+            if (dropdown) {
+                dropdown.innerHTML += 
```

Comment:
Please use `HTML.inner`

Reply:
That does something else though. Mind the `+=`. `HTML.beforeEnd` should do the trick.